src/js/edu-benefits/5490/config/form.js

```diff
@@ -91,15 +89,58 @@ const formConfig = {
       pages: {
         applicantInformation: applicantInformation(fullSchema5490, {
           labels: { relationship: relationshipLabels }
-        })
+        }),
+        additionalBenefits: additionalBenefits(fullSchema5490, {
+          fields: ['civilianBenefitsAssistance', 'civilianBenefitsSource']
+        }),
+        applicantService: {
+          title: 'Applicant service',
```

Comment:
I think the idea was to add the subheadings. From the original issue:

>Red text is a change from current copy.
>Parentheticals are subheads within a chapter

But yeah, what say ye, @goldenmeanie?